test/ecdsa_genpkey_auth.sh

```diff
@@ -6,15 +6,15 @@ set -eufx
 echo -n "abcde12345abcde12345abcde12345ab" > testdata
 
 # generate private key as PEM
-openssl genpkey -provider tpm2 -algorithm EC -pkeyopt group:P-256 \
+openssl genpkey -provider tpm2 -provider base -algorithm EC -pkeyopt group:P-256 \
```

Comment:
Is the `-provider base` really necessary here? There are no file inputs.

Reply:
Uhm...  apparently not.  Removed